spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/admin/SpannerSchemaOperations.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.spanner.core.admin;
+
+import com.google.cloud.spanner.Key;
+
+import org.springframework.cloud.gcp.data.spanner.core.convert.SpannerConverter;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerMappingContext;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerPersistentEntity;
+import org.springframework.util.Assert;
+
+/**
+ * @author Chengyuan Zhao
+ */
+public class SpannerSchemaOperations {
```

Comment:
Apologies - but I have a last rename idea: `SpannerEntityUtils` - because this is not about resource management, or exposing existing functionality. It's more like a helper class around entities, which if we wanted to we could even implement as a similar bytecode generator as Lombok does - getId() (and later getCreateDDL(), getDropDDL()) could live _on the entity class_ itself...

Reply:
oh and let's add a javadoc to it too, so that users know what this is for. Thank you!

Reply:
@balopat PTAL ! changes made